chapter11/red/Date/Oogaboogoo.pm

```diff
@@ -0,0 +1,80 @@
+package Date::Oogaboogoo;
+
+=head1 NAME
+
+Date::Oogaboogoo
+
+=head1 VERSION
+
+Version 1.0
+
+=head1 SYNOPSIS
+
+  use Date::Oogaboogo;
+
+  my $day = Date::Oogaboogoo::number_to_day_name(1); # dip
+  my $mon = Date::Oogaboogoo::number_to_month_name(3); # rod
+
+=head1 DESCRIPTION
+
+A date formatting library for display of dates in the Oogaboogoo calendar.
+
+=cut
+
+use strict;
+use warnings;
+require Exporter;
+use vars qw( @ISA @EXPORT_OK );
+
+@ISA = 'Exporter';
+@EXPORT_OK = qw(
+  number_to_day_name
+  number_to_month_name
+  );
```

Comment:
Yeah, was a little worried this might be something we have to change towards in the next chapter, but I felt uncomfortable doing it the other way.